backend/src/apiserver/client_manager/client_manager.go

```diff
@@ -35,25 +37,33 @@ import (
 )
 
 const (
-	minioServiceHost       = "MINIO_SERVICE_SERVICE_HOST"
-	minioServicePort       = "MINIO_SERVICE_SERVICE_PORT"
-	minioServiceRegion     = "MINIO_SERVICE_REGION"
-	minioServiceSecure     = "MINIO_SERVICE_SECURE"
-	pipelineBucketName     = "MINIO_PIPELINE_BUCKET_NAME"
-	pipelinePath           = "MINIO_PIPELINE_PATH"
-	mysqlServiceHost       = "DBConfig.Host"
-	mysqlServicePort       = "DBConfig.Port"
-	mysqlUser              = "DBConfig.User"
-	mysqlPassword          = "DBConfig.Password"
-	mysqlDBName            = "DBConfig.DBName"
-	mysqlGroupConcatMaxLen = "DBConfig.GroupConcatMaxLen"
-	mysqlExtraParams       = "DBConfig.ExtraParams"
-	archiveLogFileName     = "ARCHIVE_CONFIG_LOG_FILE_NAME"
-	archiveLogPathPrefix   = "ARCHIVE_CONFIG_LOG_PATH_PREFIX"
-	dbConMaxLifeTime       = "DBConfig.ConMaxLifeTime"
-
-	visualizationServiceHost = "ML_PIPELINE_VISUALIZATIONSERVER_SERVICE_HOST"
-	visualizationServicePort = "ML_PIPELINE_VISUALIZATIONSERVER_SERVICE_PORT"
+	minioServiceHost   = "MINIO_SERVICE_SERVICE_HOST"
+	minioServicePort   = "MINIO_SERVICE_SERVICE_PORT"
+	minioServiceRegion = "MINIO_SERVICE_REGION"
+	minioServiceSecure = "MINIO_SERVICE_SECURE"
+	pipelineBucketName = "MINIO_PIPELINE_BUCKET_NAME"
+	pipelinePath       = "MINIO_PIPELINE_PATH"
+
+	mysqlServiceHost       = "DBConfig.MySQLConfig.Host"
+	mysqlServicePort       = "DBConfig.MySQLConfig.Port"
+	mysqlUser              = "DBConfig.MySQLConfig.User"
+	mysqlPassword          = "DBConfig.MySQLConfig.Password"
+	mysqlDBName            = "DBConfig.MySQLConfig.DBName"
+	mysqlGroupConcatMaxLen = "DBConfig.MySQLConfig.GroupConcatMaxLen"
+	mysqlExtraParams       = "DBConfig.MySQLConfig.ExtraParams"
+
+	postgresHost     = "DBConfig.PostgreSQLConfig.Host"
+	postgresPort     = "DBConfig.PostgreSQLConfig.Port"
+	postgresUser     = "DBConfig.PostgreSQLConfig.User"
+	postgresPassword = "DBConfig.PostgreSQLConfig.Password"
+	postgresDBName   = "DBConfig.PostgreSQLConfig.DBName"
```

Comment:
The newly added Postgres configs seems so generic and completely covered by existing configs for mysql already. Do we need to separate them?

Reply:
I agree that they are pretty much the same fields here, but I think it is possible that there can be different values for the two types of DB, as well as different configs available, so it's safer to have one set of values for each of them. For example, the default host name and port number for MySQL and PostgreSQL are different.